src/autotype/AutoTypeMatchView.cpp

```diff
@@ -78,7 +78,7 @@ void AutoTypeMatchView::keyPressEvent(QKeyEvent* event)
     }
 }
 
-void AutoTypeMatchView::setMatchList(const QList<AutoTypeMatch>& matches, bool selectFirst)
+void AutoTypeMatchView::setMatchList(const QList<AutoTypeMatch>& matches, int selectedIndex, bool selectFirst)
```

Comment:
Do we need the selectFirst parameter? Just default the selectedIndex param to 0.

Reply:
Yes unless there's a better way, see my comment for @michaelk83.